### PR TITLE
add shebang line in setup_custom_pythonpath.sh.in template

### PR DESCRIPTION
**Pull request by mikaelarguedas**
connects to #69

### CODE REVIEW

**Comment by joshughes** [comment]
@mikaelarguedas this solves the issue I was seeing

**Comment by mikaelarguedas** [comment]
@ros-pull-request-builder retest this please

**Comment by joshughes** [comment]
@mikaelarguedas sorry to bother you but was curious how we might know when this fix is available upstream 

Now that its merged will a new package be generated right away or does the ROS team have a release cadence they adhere to 

Currently we pull from http://packages.ros.org/ros/ubuntu/pool/main/r/ros-kinetic-rqt-reconfigure/

**Comment by mikaelarguedas** [comment]
TL:DR I'd say about 3 weeks from now

The process for getting a package available:
- making a release
- the ROS buildfam will build the package and make it available in the [testing repository](http://wiki.ros.org/ShadowRepository)
- every few weeks (usually 2 weeks but sometimes a bit more) we sync the testing repository to the main repository (the one you are using).

For packages that low in the stack we try to synchronize the releases because it results in rebuild almost the entire ROS eco-system. I expect us to make a new release of ros "core" packages around the beginning of next week, we will then leave them in testing for ~2weeks to track and address possible regressions before making them available on the main repository.
